csv/result_test.go

```diff
@@ -1420,6 +1423,53 @@ func TestMultiResultDecoder(t *testing.T) {
 	}
 }
 
+func TestTable(t *testing.T) {
+	executetest.RunTableTests(t, executetest.TableTest{
```

Comment:
These changes make it so the CSV decoder uses an allocator, does this test make sure that all memory is returned when the table is released?

Reply:
Yes. The `RunTableTests` method checks that the allocator has both been used and that it is currently at zero allocations.